portfolio/src/main/webapp/nav.js

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// DIV_IDs
+
+/** 
+* The id of the div where the nav bar will be added.
+* @const {String}
+*/
+const NAV_ID = "my-nav";
+
+/** 
+* list containing JS objects with link information. link fields correspond to anchor fields
+* @const {Object List}
+*/
+const LINKS = [
+    {
+      target:"_blank", 
+      href:"mailto: coa22@cornell.edu",
+      class:"nav_bar", 
+      text:"/email_me "
+    },
+    {
+      target:"_blank", 
+      href:"https://github.com/Obinnabii",
+      class:"nav_bar", 
+      text:"/see_github "
+    },
+    {
+      target:"_blank", 
+      href:"https://drive.google.com/file/d/1tc2YaWmPzVhdCSb3Gv6VVz-599Qa5CrT/view?usp=sharing",
+      class:"nav_bar", 
+      text:"/see_resume "
+    },
+    {
+      target:"_self", 
+      href:"index.html",
+      class:"nav_bar", 
+      text:"/home_page "
+    },
+    {
```

Comment:
I was going to do that but I made it a function so that I wouldn't have to make changes to the nav over multiple files. like if I wanted to add a new tab, I would only do it in the nav.js file and not have to make changes to comments.html or any other webpages I add.